spec/cypress/support/commands.js

```diff
@@ -72,6 +74,50 @@ Cypress.Commands.add("verifySignInEmailSent", (user) => {
   });
 });
 
+Cypress.Commands.add("verifySignInEmailSentForEmail", (email) => {
```

Comment:
I'm not sure about having these in the commands file as they're only being used in one place

Reply:
Would you move those commands into another file? I do like having cypress commands - they do make the code better

Reply:
I'd probably put the code directly in the step definition, I'm not sure what the advantage of splitting it across multiple files is when it isn't being reused!

Reply:
The problem with moving the code around was that it depends in one of the functions in commands.js for computing email headers. Also other commands in commands.js also are being used in one place. How would you address this in terms of dependency on computeHeadersFromEmail function.

Reply:
The step definitions are also defined in a single file :)

Reply:
Done